CHANGES.rst

```diff
@@ -25,6 +25,6 @@
 Changes
 =======
 
-Version 1.0.0 (released 2016-01-07)
+Version 1.0.0 (released 2016-02-05)
```

Comment:
You mean if we add ``1.0.0`` -> ``1.0.0a5``? I my opinion no. I'm only fixing it the packages where it's already there.